cloudant-client/src/main/java/com/cloudant/client/api/views/SettableViewParameters.java

```diff
@@ -52,6 +52,53 @@
      */
     String STALE_UPDATE_AFTER = "update_after";
 
+    /**
+     * Constant for the value "false" for use with
+     * {@link com.cloudant.client.internal.views.CommonViewRequestBuilder#stable(String)}
+     * <P>
+     * false: Do not prefer view results from a 'stable' set of shards.
+     * </P>
+     */
+    String STABLE_FALSE = "false";
+
+    /**
+     * Constant for the value "true" for use with
+     * {@link com.cloudant.client.internal.views.CommonViewRequestBuilder#stable(String)}
+     * <P>
+     * true: Prefer view results from a 'stable' set of shards. The results are from a view that is
+     * less likely to be updated soon.
+     * </P>
+     */
+    String STABLE_TRUE = "true";
+
+    /**
+     * Constant for the value "false" for use with
+     * {@link com.cloudant.client.internal.views.CommonViewRequestBuilder#update(String)}
+     * <P>
+     * false: Return results before updating the view.
+     * </P>
+     */
+    String UPDATE_FALSE = "false";
```

Comment:
It's not a big deal, but could use `Boolean.FALSE.toString()` or `Boolean.toString(false)` here and equivalently for the true case.

Reply:
Done in 1dceeae3f4050abe9c6909eb1cb4df9be959f62b